lib/tasks/add_ost_apps.rake

```diff
@@ -11,32 +11,35 @@ namespace :accounts do
     task :create_apps, [:app_domain_suffix, :admin_password] => :environment do |t, args|
       ActiveRecord::Base.transaction do
         begin
+          # Get the application url suffix if provided
+          suffix = args[:app_domain_suffix] || ''
+          # Get the admin password
+          password = args[:admin_password]
           # Create application owner if needed
           app_owner_group = Group.find_or_create_by_name('ost_app_admin_group')
-          user = User.find_or_create_by_username('ost_app_admin')
-          app_owner_group.add_owner(user)
-          password = args[:admin_password]
-          user.identity = Identity.create do |identity|
-            identity.password = password
-            identity.password_confirmation = password
-            identity.user_id = user.id
+          user = User.find_or_create_by_username('ost_app_admin') do |new_user|
```

Comment:
Can you add some comments (a few sentences is fine) to the top just to give a quick summary of what it is this file is doing?  I was wondering why we were making an admin user in Accounts, and it took me a while to realize this user is just the owner of the apps being created.  And just a sentence to say what the overall purpose of this file is.  Thanks!